examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java

```diff
@@ -260,8 +261,7 @@ public static PipelineResult run(DataTokenizationOptions options) {
         rows = rows.apply(Window.into(FixedWindows.of(parseDuration(options.getWindowDuration()))));
       }
     } else {
-      throw new IllegalStateException(
-          "No source is provided, please configure File System or Pub/Sub");
+      throw new IllegalStateException("No source is provided, please configure GCS or Pub/Sub");
```

Comment:
Why is it only GCS? Can't we support any FS in Beam?

Reply:
done